app/javascript/packs/loader.js

```diff
@@ -0,0 +1,16 @@
+class Loader {
+  constructor(element){
+    this.parent = element;
+    this.template = '<i id="loader" class="fas fa-circle-notch fa-spin"></i>';
```

Comment:
Can we aggree on a convention that use backticks for html templates in JS ?
```suggestion
    this.template = `<i id="loader" class="fas fa-circle-notch fa-spin"></i>`;
```